### ISSUE TITLE
Remote Manager bugfix and problems with the config files

### ISSUE THREAD

**Issue by rbeignon** [PyMoDAQ/PyMoDAQ#435]
I needed to modify the line 20 of utils/managers/remote_manager.py from:
`actuator_actions = ['move_Rel', 'move_Rel_p', 'move_Rel_m']`
to:
`actuator_actions = ['move_rel', 'move_rel_p', 'move_rel_m']`
in order to make it work. It seems that it hasn't been updated with the v4.
There are more problems with the modifying and loading new config. I need to restart PyMoDAQ to apply the new config and adding a new module to control while editing sometimes get stuck.

**Comment by seb5g** [comment]
> I needed to modify the line 20 of utils/managers/remote_manager.py from:
> `actuator_actions = ['move_Rel', 'move_Rel_p', 'move_Rel_m']`
> to:
> `actuator_actions = ['move_rel', 'move_rel_p', 'move_rel_m']`
> in order to make it work. It seems that it hasn't been updated with the v4.

could you propose a pull request to make it work again? Do you know how to do so?

Could you be more specific about the config? What config, when doing what?

**Comment by rbeignon** [comment]
Ok I created the pull request for the actuator_actions.

For the problem with the config:
When I want to load a config once a previous config has already been loaded I get this message:
`File "C:\Users\rbeignon\.conda\envs\pmd4\lib\site-packages\pyqtgraph\parametertree\Parameter.py", line 628, in insertChild`
`raise ValueError("Already have child named %s" % str(name))`
`ValueError: Already have child named shortcut00`

When I edit a remote config file, if I try to add a new actuator I get this error:
`File "C:\Users\rbeignon\.conda\envs\pmd4\lib\site-packages\pymodaq\utils\managers\remote_manager.py", line 50, in <listcomp>
    child_indexes = [int(par.name()[len(name_prefix) + 1:]) for par in self.children()]`
`ValueError: invalid literal for int() with base 10: ''`

When editing, the "Set Shortcut" button doesn't activate the window to press a key

**Comment by seb5g** [comment]
> For the problem with the config:
> When I want to load a config once a previous config

which config are you talking about. The one specifying the remote things? when you say load a config once a previous config, do you mean loading an other remote file from the menu bar?

**Comment by rbeignon** [comment]
Yes, the one specifying the remote things. It can be loading a different one from the menu bar or the same one that I modified. I need to restart PyMoDAQ to have the new config applied

**Comment by seb5g** [comment]
Note for this issue:

* first part is fixed by #438 
* but for this not to append again, one should encapsulate the method names in an enum or equivalent with a test for the existence of such method

**Comment by seb5g** [comment]
ok we should probably implement a clear method to empty the current configuration before allowing to load a new one.

**Comment by malik-irain** [comment]
Will be handled with ManagerBase refactorization for Roi and Remote managers
